### ISSUE TITLE
List search doesn't communicate when search completed and result number to assistive technologies (3.3.2)

### ISSUE THREAD

**Issue by Blackbaud-MattGregg** [blackbaud/skyux-list-builder#93]
### Issue from RE NXT accessibility audit 2019

The search results are not informated to screen reader users.
For example, on the Portfolio page, When a user searches items using the search field, the search takes place and the number of items in the list are updated on the top of the page but screen reader users are not informed.

### Impact
The screen reader users remain completely unaware that search is completed and the content on the page is updated.

### Expected Behavior
Developers must provide a mechanism that screen readers are informed when the search is completed and number of records are updated on the page.
This can be done using aria-live region or moving focus to the number of search results found.
[Matt note]: one idea could be having off-screen element in live region that updates with count (e.g., 10 items found) built in so teams wouldn't have to connect to their list count display for this.
